lib/fittings/swagger_render.js

```diff
@@ -6,6 +6,7 @@ var injector = require('../../index.js').injector;
 var _ = injector.get('_');
 
 module.exports = function create(fittingDef) {
+    injector = require('../../index.js').injector;
```

Comment:
line #5 has the same code, what's the difference? Could you please explain more why this is the root case?